src/common/common.gyp

```diff
@@ -0,0 +1,26 @@
+{
+  'includes':[
+    'common.gypi',
+  ],
+  'targets': [{
+    'target_name': 'tizen_common',
+    'type': 'static_library',
+    'sources': [
+      'extension.cc',
+      'extension.h',
+      'logger.cc',
+      'logger.h',
+      'picojson.h',
+      'scope_exit.h',
+      'utils.h',
+      'XW_Extension.h',
+      'XW_Extension_EntryPoints.h',
+      'XW_Extension_Permissions.h',
+      'XW_Extension_Runtime.h',
+      'XW_Extension_SyncMessage.h',
+    ],
```

Comment:
I was trying to switch flags in few different ways, without success. The main problem is that [additional build targets](https://github.com/crosswalk-project/tizen-extensions-crosswalk/blob/master/src/media_renderer/media_renderer.gyp#L37) (which are [disabling -std=c++0x](https://github.com/crosswalk-project/tizen-extensions-crosswalk/blob/master/src/media_renderer/media_renderer.gyp#L176)) in plugins are [including common.gypi](https://github.com/crosswalk-project/tizen-extensions-crosswalk/blob/master/src/media_renderer/media_renderer.gyp#L3), which contains sources list. This target is at the bottom of inclusion tree so it's forcing to recompile src/common without C++11.